demo/Makefile

```diff
@@ -0,0 +1,82 @@
+
+# Providers: virtualbox, vmware_fusion, aws
+PROVIDER := virtualbox
+
+# Domain to use when starting flynn
+CLUSTER_DOMAIN := demo.localflynn.com
+CLUSTER := demo-${PROVIDER}
+
+# Location to install flynn CLI bin
+CLI_INSTALL_BIN := /usr/local/bin/flynn
+
+
+.PHONY: init install-cli bootstrap up down halt update ssh reset token destroy dashboard ssh-cmd
+
+
+# Initialize a new cluster
+init:
+	@$(MAKE) up
+	@$(MAKE) bootstrap
+	flynn key add
+	@printf "\nFlynn is now running in the VM.\nOpen dashboard: make dashboard\n\n"
+
+# Install flynn cli on host
+install-cli:
+	curl -sL -A "`uname -sp`" https://dl.flynn.io/cli | zcat >${CLI_INSTALL_BIN}
+	chmod +x ${CLI_INSTALL_BIN}
+
+# Bootstrap the cluster
+# This should only be called once per cluster
+# Runs bootstrap in VM and cluster add on host
+bootstrap:
+	@$(MAKE) CMD='CLUSTER_DOMAIN="${CLUSTER_DOMAIN}" flynn-host bootstrap /etc/flynn/bootstrap-manifest.json 2>&1' ssh-cmd | tee bootstrap.log
+	@(grep '^flynn cluster add .*$$' bootstrap.log || echo exit 1) | sed "s/ default / ${CLUSTER} /" | sed "s/$$(printf '\r')\$$/ --force --default/" | bash
+	@rm bootstrap.log
+
+# Start the VM and flynn
+# See https://github.com/mitchellh/vagrant/issues/4367 for use of SSH_AUTH_SOCK=''
+up:
+	CLUSTER_DOMAIN="${CLUSTER_DOMAIN}" SSH_AUTH_SOCK='' vagrant up --provider ${PROVIDER}
+	@$(MAKE) CMD='sudo start flynn-host || echo flynn-host already running or unable to start' ssh-cmd
+
+# Stop the VM
+down:
+	vagrant halt
+
+halt: down
+
+# Update the VM image to the latest flynn vagrant box
+update:
+	vagrant box update
+
+# SSH into VM
+ssh:
+	vagrant ssh
+
+# Destroy and rebuilt the VM
+reset:
+	@$(MAKE) destroy
+	@$(MAKE) update
+	@$(MAKE) init
+
+# Destroy VM
+# Run this to delete the VM and free up disk space
+destroy:
+	vagrant destroy -f
+	flynn cluster remove ${CLUSTER}
+
+# Get token for API and dashboard
+token:
+	@flynn -a dashboard env get LOGIN_TOKEN
+
+# Open dashboard in browser
+dashboard:
+	@echo Login token:
+	@$(MAKE) token
+	@sleep 1 # Allow user to see token output before browser opens
+	@open http://dashboard.${CLUSTER_DOMAIN}
+
+# Run a command in the VM via SSH
+ssh-cmd:
+	vagrant ssh -c '${CMD}'
```

Comment:
`SSH_AUTH_SOCK` needs to be specified here and in the `ssh` target. Might make sense to make a variable.